### ISSUE TITLE
[Bug] incorrect file size units on website list

### ISSUE THREAD

**Issue by devenbj** [hestiacp/hestiacp#3540]
### Describe the bug

In my users account, under the web view (/list/web/), it shows the disk space occupied by each domain. I can confirm that most of them are accurate +/- 5%. There is one exception. Most of the sites are between 4 and 450 mb. I have one site that is 1002 mb. That one site shows up as 0.98 mb. If that were to say 0.98 gb, it would be accurate. I suspect the code that adds the units to the end of the disk size is not working correctly.

### Tell us how to replicate the bug

1. log in as admin
2. click on a user with websites
3. compare the disk size with the actuals...
4. all sites show an "mb" size measurement, even though one should be reporting "gb"

### Which components are affected by this bug?

Control Panel Web Interface

### Hestia Control Panel Version

1.7.4

### Operating system

Ubuntu 20.04

**Comment by devenbj** [comment]
I think I've identified the issue.

In the file /web/inc/main.php
function humanize_usage_size; lines: 347, 349, 351 use 1024 as the comparator; lines: 355, 361, 367 use 1000 as the comparator
function humanize_usage_measure; lines: 384, 386, 388, 390, 393 use 1024 as the comparator

The issue likely only affect disk sizes between 1,000 and 1,023 (mb, gb, or tb).

I changed the 1024 value to 1000 for the human_usage_measure function in lines 388, 390, 393. This resolved the issue for me. It now reports 0.98 gb
